core/tests/ts-integration/tests/base-token.test.ts

```diff
@@ -26,6 +26,24 @@ describe('base ERC20 contract checks', () => {
         isETHBasedChain = zksync.utils.isAddressEq(baseToken, zksync.utils.ETH_ADDRESS_IN_CONTRACTS);
     });
 
+    test('Base token ratio is updated on L1', async () => {
+        if (isETHBasedChain) {
+            return;
+        }
+
+        let govMnemonic = ethers.Mnemonic.fromPhrase(require('../../../../etc/test_config/constant/eth.json').mnemonic);
+        let govWalletHD = ethers.HDNodeWallet.fromMnemonic(govMnemonic, "m/44'/60'/0'/0/1");
+        const adminGovWallet = new ethers.Wallet(govWalletHD.privateKey, alice._providerL1());
```

Comment:
it seems you don't need `adminGovWallet` in this test, `alice` is enough

Reply:
Good point - changed

Reply:
Actually alice doesn't seem to have enough permissions to call these functions, reverted back

Reply:
why? AFAICT It's view getter so it doesn't matter which wallet to connect, you can even connect none

Reply:
Yes, you are right - I've been passing in a wrong wallet. Changed to use `alice`